airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryRecordConsumer.java

```diff
@@ -134,7 +137,11 @@ protected JsonNode formatRecord(final Schema schema, final AirbyteRecordMessage
     // use BQ helpers to string-format correctly.
     final long emittedAtMicroseconds = TimeUnit.MICROSECONDS.convert(recordMessage.getEmittedAt(), TimeUnit.MILLISECONDS);
     final String formattedEmittedAt = QueryParameterValue.timestamp(emittedAtMicroseconds).getValue();
-    final JsonNode formattedData = StandardNameTransformer.formatJsonPath(recordMessage.getData());
```

Comment:
Don't we also need to add tests for this? For now I was abelt to see changes in tests for the bigquery-denormalized connector. Did I miss something? 
Gcs uploading mode is used in both bigquery and bigquery-denormalized connectors.
Only Gcs uploading mode was affected, right? Don't we also need to check it for direct uploading? Not sure if it's needed, just asking :)

Reply:
Looks like we don't need to have such DateTime handling for BigQuery standard connector. And the code responsible for `format` field handling is not required. The reason because we have BigQuery schema hardcoded, see:
```java
private static final com.google.cloud.bigquery.Schema SCHEMA = com.google.cloud.bigquery.Schema.of(
      Field.of(JavaBaseConstants.COLUMN_NAME_AB_ID, StandardSQLTypeName.STRING),
      Field.of(JavaBaseConstants.COLUMN_NAME_EMITTED_AT, StandardSQLTypeName.TIMESTAMP),
      Field.of(JavaBaseConstants.COLUMN_NAME_DATA, StandardSQLTypeName.STRING));

```
and
```java
 protected Schema getBigQuerySchema(final JsonNode jsonSchema) {
    return SCHEMA;
  }
```
Another situation for BigQuery denormalized connector:
```java
@Override
  protected Schema getBigQuerySchema(final JsonNode jsonSchema) {
    final List<Field> fieldList = getSchemaFields(getNamingResolver(), jsonSchema);
    if (fieldList.stream().noneMatch(f -> f.getName().equals(JavaBaseConstants.COLUMN_NAME_AB_ID))) {
      fieldList.add(Field.of(JavaBaseConstants.COLUMN_NAME_AB_ID, StandardSQLTypeName.STRING));
    }
    if (fieldList.stream().noneMatch(f -> f.getName().equals(JavaBaseConstants.COLUMN_NAME_EMITTED_AT))) {
      fieldList.add(Field.of(JavaBaseConstants.COLUMN_NAME_EMITTED_AT, StandardSQLTypeName.TIMESTAMP));
    }
    return com.google.cloud.bigquery.Schema.of(fieldList);
  }

  private static List<Field> getSchemaFields(final BigQuerySQLNameTransformer namingResolver, final JsonNode jsonSchema) {
    Preconditions.checkArgument(jsonSchema.isObject() && jsonSchema.has(PROPERTIES_FIELD));
    final ObjectNode properties = (ObjectNode) jsonSchema.get(PROPERTIES_FIELD);
    return Jsons.keys(properties).stream().map(key -> getField(namingResolver, key, properties.get(key)).build()).collect(Collectors.toList());
  }
```